cazador/src/App.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react';
+import './App.css';
+import { withRouter, Switch, Route, NavLink } from 'react-router-dom';
+import RegisterForm from './components/RegisterForm';
+// import UsersList from './components/UsersList';
+import LoginForm from './components/LoginForm';
+import axios from 'axios';
+
+
+const url = process.env.REACT_APP_API_URL;
+
+class App extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      loggedIn: false,
+      users: [],
+    };
+  }
+
+
+  //logout 
+  removeToken = () => {
+    //delete token
+    window.localStorage.removeItem('my_token');
+    //set loggedin to false
+    this.setState({loggedIn:false});
```

Comment:
you should probably clear out the users too here, since that is "privileged" information